audit_log/tests/api/test_views.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+from dateutil import parser
+from django.urls import reverse
+from rest_framework import status
+
+from audit_log.models import AuditLog
+from users.tests.factories import ProfileFactory
+from users.tests.utils import _create_token
+
+_common_fields = {
+    "audit_event": {
+        "origin": "APARTMENT_APPLICATION_SERVICE",
+        "status": "SUCCESS",
+        "date_time_epoch": 1590969600000,
+        "date_time": "2020-06-01T00:00:00.000Z",
+        "actor": {
+            "role": "OWNER",
+            "profile_id": "73aa0891-32a3-42cb-a91f-284777bf1d7f",
+        },
+        "operation": "READ",
+        "target": {
+            "id": "73aa0891-32a3-42cb-a91f-284777bf1d7f",
+            "type": "Profile",
+        },
+    }
+}
+
```

Comment:
Could we add a test for non authorized users here as well.

Reply:
Added test for non authorized users